.github/actions/setup-minikube/action.yaml

```diff
@@ -8,7 +8,7 @@ runs:
     run: |
       set -x
       sudo apt-get install -y --no-install-recommends conntrack
-      sudo curl --fail -L https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 -o /usr/local/bin/minikube
+      sudo curl --fail -L https://storage.googleapis.com/minikube/releases/v1.22.0/minikube-linux-amd64 -o /usr/local/bin/minikube
```

Comment:
nit: version could be extracted to env variable, similar to `go_version`

Reply:
it would have to be an argument for it to be able to reuse outside this library file (`go-version` arg vs. the `go_version` variable) but given this action is dedicated for our repo and we use it with just one version for now I'd postpone that to when we actually need it